libs/market-info/src/components/market-info/info-market.tsx

```diff
@@ -97,7 +98,13 @@ export const Info = ({ market, onSelect }: InfoProps) => {
     [market]
   );
   const { data: asset } = useAssetDataProvider(assetId ?? '');
+
   if (!market) return null;
+
+  const marketAccounts = compact(market.accountsConnection?.edges).map(
```

Comment:
thought (non-blockng): This is becoming something we do so often would it be worth adding a helper function for it?

Reply:
That would make sense, but multiple accounts queries with different but overlapping resulting types makes this not really possible, at least not without additional code to handle different query shapes. Going to resolve but LMK if you have a good solution.